### ISSUE TITLE
Bundle error upgrading to current version

### ISSUE THREAD

**Issue by pragunvohra** [intercom/intercom-cordova#63]
I'm having trouble upgrading intercom plugin versions from 1.1.4 to the current version.

## Version info
  - Intercom for Cordova/PhoneGap version: 1.1.4 -> current
  - iOS/Android OS version: iOS 8+, Android 4.4+
  - iOS/Android Device: iPhone 6S+, Nexus 5X

## Expected behavior
Successful installation

## Actual behavior
It seems to just be successfully installing on android, but failing on iOS.

## Steps to reproduce
 1. cordova plugin remove cordova-plugin-intercom
 2. cordova plugin add cordova-plugin-intercom

## Logs
```
pragun$ cordova plugin remove cordova-plugin-intercom
Uninstalling cordova-plugin-intercom from android
Uninstalling cordova-plugin-intercom from ios
Removing "cordova-plugin-intercom"
pragun$ cordova plugin add cordova-plugin-intercom
Fetching plugin "cordova-plugin-intercom" via npm
Installing "cordova-plugin-intercom" for android
Installing "cordova-plugin-intercom" for ios
Error during processing of action! Attempting to revert...
Failed to install 'cordova-plugin-intercom':Error: Uh oh!
cannot find "plugins/cordova-plugin-intercom/src/ios/Intercom.framework/Resources/Intercom.bundle" ios <resource-file>
```

**Comment by jtreanor** [comment]
@pragunvohra Thanks for reporting.

This is now fixed in 3.0.2 🎉

**Comment by pragunvohra** [comment]
thanks @jtreanor i'll give it a go again!

**Comment by pragunvohra** [comment]
Just tried @jteanor, now I'm receiving the following msg in xcode:

#import <Intercom/Intercom.h>

`'Intercom.h' file not found`

**Comment by jtreanor** [comment]
@pragunvohra What version of Xcode and Cordova are you using?

Can you post the full error you are getting for `'Intercom.h' file not found`

**Comment by pragunvohra** [comment]
I'm using XCode 7.3 and Cordova 5.2.0

**Comment by jtreanor** [comment]
Note you will need to use ios version 4.0.0.

We have just released [3.0.3](https://github.com/intercom/intercom-cordova/releases/tag/3.0.3) which I believe should resolve this issue.
